.github/codecov.yml

```
@@ -0,0 +1,8 @@
+component_management:
+  individual_components:
+    - component_id: "contentauthor"
+      paths:
+        - "sourcecode/apis/"
```

Per [the documentation](https://docs.codecov.com/docs/components), this seems like it should be
```suggestion
        - "sourcecode/apis/contentauthor/**"
```